src/main/webapp/app/exam/participate/exam-participation.service.ts

```diff
@@ -66,6 +66,20 @@ export class ExamParticipationService {
         return this.getStudentExamFromServer(url, courseId, examId);
     }
 
+    /**
+     * Retrieves the exercise ids of the user exam of the current user from server or localstorage.
+     * @param courseId the id of the course the exam is created in
+     * @param examId the id of the exam
+     */
+    public loadStudentExamExerciseIds(courseId: number, examId: number): Observable<number[]> {
+        const url = this.getResourceURL(courseId, examId) + '/student-exams/exercise-ids';
+        return this.httpClient.get<number[]>(url).pipe(
+            map((exerciseIds: number[]) => {
+                return exerciseIds;
+            }),
```

Comment:
Looks like a redundant map stage to me. Could we remove it?

Reply:
Yes, great find 👍